conn2res/task.py

```diff
@@ -99,21 +142,40 @@ def classification(x, y, **kwargs):
     return score
 
 
-def multiClassClassification(x, y, **kwargs):
+def multiClassClassification(x, y, model=None, **kwargs):
     """
     Multi-class Classification tasks
-    #TODO
+    # TODO
     """
 
+    # pop variables from kwargs
+    sample_weight_train, sample_weight_test = kwargs.pop(
+        'sample_weight', (None, None))
+
     x_train, x_test = x
     y_train, y_test = y
 
-    # capture only decision time points
-    idx_train = np.nonzero(y_train)
-    idx_test  = np.nonzero(y_test)
+    # specify default model
+    if model is None:
+        model = OneVsRestClassifier(RidgeClassifier(
+            alpha=0.0, fit_intercept=False, **kwargs))
```

Comment:
Again, as we discussed in our meeting, OneVsRestClassifier is actually not needed when used with RidgeClassifier as RidgeClassifier has the capability to perform multi-class classification off the shelf and gives the same result.

Another drawback of using OneVsRestClassifier is that is does not support sample weights. In contrast, had we used RidgeClassifier, it supports sample weights.